src/java.compiler/share/classes/javax/tools/SimpleJavaFileObject.java

```diff
@@ -136,7 +137,8 @@ public CharSequence getCharContent(boolean ignoreEncodingErrors) throws IOExcept
     /**
      * {@inheritDoc FileObject}
      * @implSpec
-     * This implementation wraps the result of openOutputStream in a Writer.
+     * This implementation wraps the result of {@link
+     * openOutputStream} in a {@link Writer}.
```

Comment:
```suggestion
     * This implementation wraps the result of {@linkplain #openOutputStream}
     * in a {@link Writer}.
```

Reply:
Still missing the `#` to make a valid link:
```suggestion
     * This implementation wraps the result of {@link #openOutputStream}
     * in a {@link Writer}.
```

Reply:
Hmm. Well, the build and docs build succeed with the `#` and produce a link in the generated HTML, but I'll add it before pushing. Thanks,